sdks/python/apache_beam/typehints/trivial_inference.py

```diff
@@ -303,15 +311,23 @@ def infer_return_type_func(f, input_types, debug=False, depth=0):
   last_pc = -1
   while pc < end:
     start = pc
-    op = ord(code[pc])
-
+    if sys.version_info[0] == 2:
+      op = ord(code[pc])
+    else:
+      op = code[pc]
     if debug:
       print('-->' if pc == last_pc else '    ', end=' ')
       print(repr(pc).rjust(4), end=' ')
       print(dis.opname[op].ljust(20), end=' ')
+
     pc += 1
     if op >= dis.HAVE_ARGUMENT:
-      arg = ord(code[pc]) + ord(code[pc + 1]) * 256 + extended_arg
+      if sys.version_info[0] == 2:
+        arg = ord(code[pc]) + ord(code[pc + 1]) * 256 + extended_arg
+      elif sys.version_info[0] == 3 and sys.version_info[1] < 6:
+        arg = code[pc] + code[pc + 1] * 256 + extended_arg
+      else:
+        pass # TODO(luke-zhu): Python 3.6 bytecode to wordcode changes
```

Comment:
<!--new_thread; commit:b015963f671ceb39f8033014eb6d4e46a4a734bc; resolved:0-->
Do we need to test it for 2.x, < 3.6 and >=3.6?

What are the usage numbers for different python versions? Can we support only python >= 3.6

cc: @robertwb

Reply:
Python 3.6 is less than a year old, probably should consider supporting 3.5 at least.

Reply:
The changes work for up to Python 3.5. If we ever do want to support 3.6+, we'll need to change certain increments of the pc variable from 1 to 2. I'll look into it this weekend.